### ISSUE TITLE
Migrate CI out from Travis-CI

### ISSUE THREAD

**Issue by Davidy22** [Guake/guake#1876]
Travis-ci changed from travis-ci.org to travis-ci.com. In june this year, they finally cut off their .org site, which is the reason why [there are no builds of guake in the last 3 months](https://travis-ci.org/github/Guake/guake/builds) and all the pull request build results show permanently in-limbo build statuses. Requesting someone with settings permissions to do the migration (like @gsemet) to [migrate the repo to the new .com](https://docs.travis-ci.com/user/migrate/open-source-repository-migration#migrating-a-repository), as merges require passing CI builds which is blocked by CI not happening.

**Comment by gsemet** [comment]
Isn’t it possible to switch to GitHub action ?

**Comment by mlouielu** [comment]
I'll second to move out from Travis to anywhere else. maybe we can checkout
GitHub Action first.

Gaetan Semet ***@***.***> 於 2021年8月31日 週二 22:27 寫道：

**Comment by Davidy22** [comment]
Oh, if we want to switch to github actions I'll take a shot at recreating the travis CI

**Comment by mlouielu** [comment]
I've label this to `priority: URGENT` as I would like to see CI come back before we accept any new PR.

**Comment by mlouielu** [comment]
#1877 take the first step to migrate to GitHub Actions.

**Comment by Davidy22** [comment]
We've done the move and it works pretty nicely, so I'm marking this as resolved.
